projects/kit/components/input-phone-international/input-phone-international.options.ts

```diff
@@ -1,15 +1,29 @@
 import {InjectionToken, ValueProvider} from '@angular/core';
+import {TuiNumberHandler, TuiStringHandler} from '@taiga-ui/cdk';
 import {TuiCountryIsoCode} from '@taiga-ui/i18n';
+import {tuiGetMaxAllowedPhoneLength, tuiIsoToCountryCode} from '@taiga-ui/kit/utils';
 
 export interface TuiInputPhoneInternationalOptions {
     readonly countries: readonly TuiCountryIsoCode[];
     readonly countryIsoCode: TuiCountryIsoCode;
+    readonly allowedPhoneLength: TuiNumberHandler<{
```

Comment:
I don't think these make sense in options. Options are countries, the rest is not a concern of the developer providing options.